src/js/common/schemaform/SchemaForm.jsx

```diff
@@ -88,6 +88,7 @@ class SchemaForm extends React.Component {
   }
 
   onBlur(id) {
+    _.debounce(1000, this.props.autoSave);
```

Comment:
This isn't doing anything right now, it returns a function you can call that will be debounced.

I think it's slightly better if instead of passing an `autoSave` prop we pass an `onBlur` one, and do the debouncing/saving all in FormPage, which keeps it all in one place.

So in here you would have something like `this.props.onBlur(id);`.

And in `FormPage` you can put `this.autoSaveDebounced = _.debounce(1000, this.autoSave)` in the constructor, and pass `onBlur={this.autoSaveDebounced}` in the props passed to `SchemaForm`.